README.md

```diff
@@ -30,36 +32,36 @@ Im Body wird eine Payload als JSON in folgendem Format verschickt:
 }
 ```
 
-Zu den Attributen im Detail:
+More details to the attributes:
 
 - `vorgangsnummer`:
 
-  Sowohl bei Änderungen im Vorgang als auch im Antrag wird in einer KEX-PUSH Message stets nur die Vorgangsnummer übermittelt.
+  For changes in either Vorgang or Antrag the KEX-PUSH message always only contains the Vorgangsnummer.
 
 - `kundenbetreuerPartnerbaum`:
 
-  Wegen der [Limitierung des Topics](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#iot-protocol-limits) auf maximal 7 Ebenen kann je nach Tiefe der Partnerstruktur ein Topic abgeschnitten werden. Um dem Subscriber dennoch Auskunft über den aktuellen Kundenbetreuer geben zu können, enthält der `kundenbetreuerPartnerbaum` die vollständige Partnerstruktur.
+  Because of the [limits for Topics](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#iot-protocol-limits) to a maximium of 7 it can happen that a topic gets cut off if the corresponding Partner tree is too deep. To still be able to notify the subscriber about a Kundenbetreuer, the property `kundenbetreuerPartnerbaum` contains the full tree.
```

Comment:
```suggestion
  Because of the [limits for Topics](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html#iot-protocol-limits) to a maximium of 7 it can happen that a topic gets cut off if the corresponding Partner tree is too deep. To still be able to notify the subscriber about a Kundenbetreuer, the property `kundenbetreuerPartnerbaum` contains the complete tree.
```